ex_feels/lib/ex_feels_web/templates/layout/app.html.eex

```diff
@@ -9,6 +9,86 @@
 
     <title>BitFeels</title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+    <style type="text/css">
+      @import url(//fonts.googleapis.com/css?family=Open+Sans:400,700);
+
+      svg {
+        font: 14px 'Open Sans';
+      }
+
+      .axis path,
```

Comment:
Lets get all this chart specific css placed into its own file then compiled with our other assets. I'd suggest `/assets/css/chart.scss` then `@import "chart";` in `app.scss`.  Then you can test the compilation by `brunch build -d` inside the `/assets` folder.

Reply:
done